net.mediaarea.MediaInfo.json

```diff
@@ -0,0 +1,82 @@
+{
+  "app-id": "net.mediaarea.MediaInfo",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.26",
+  "branch": "stable",
+  "sdk": "org.gnome.Sdk",
+  "command": "mediainfo-gui",
+  "rename-desktop-file": "mediainfo-gui.desktop",
+  "rename-appdata-file": "mediainfo-gui.metainfo.xml",
+  "rename-icon": "mediainfo",
+  "finish-args": [
+    "--filesystem=host",
```

Comment:
Do you actually need permissions that are this wide? If the scope of the app is just media files it'd be better to scope this down.

Reply:
Thanks for the review.

I don't know what to apply as permissions. Big media files are frequently stored outside /home directory.

Reply:
Yeah, I had a look at the [screenshots](https://mediaarea.net/en/MediaInfo/Screenshots) and based on that you do probably want the wide permissions. If you just open things by the file chooser then you shouldn't need any permissions at all, but given the UI, feel free to ignore the request.

Reply:
> If you just open things by the file chooser then you shouldn't need any permissions at all

MediaInfo can also open file by command line, like:
`flatpak run net.mediaarea.MediaInfo ~/MediaArea/Samples.avidv.avi`

Reply:
Flatpaks aren't really designed to be ran by users on the CLI but it can work:

`flatpak run --file-forwarding net.mediaarea.MediaInfo @@ ~/MediaArea/Samples.avidv.avi @@`